":read :cmd" reads Ex command ":cmd" output (#30628)
neovim/neovim: runtime/doc/insert.txt

							*:r!* *:read!*
:[range]r[ead] [++opt] !{cmd}
			Execute shell {cmd} and insert its standard output
			below the cursor or the specified line.  A temporary
			file is used to store the output of the command which
			is then	read into the buffer.  'shellredir' is used to
			save the output of the command, which can be set to
			include stderr or not.  {cmd} is executed like with
			":!{cmd}", any '!' is replaced with the previous
			command |:!|. See |++opt| for the possible values of
			[++opt].

							*:r:* *:read:*
:[range]r[ead] :{cmd}

> 			Execute vim {cmd} and insert its output below the

Reviewer: ```suggestion
			Execute Ex command {cmd} and insert its output below the
```

> 			specified line

Reviewer: ```suggestion
			specified line.
```

> 			Execute vim {cmd} and insert its output below the
> 			specified line

Reviewer: ```suggestion
			Execute Ex command {cmd} and insert its output below
			the cursor or the specified line.
```

							*:r!* *:read!*
:[range]r[ead] [++opt] !{cmd}
			Execute shell {cmd} and insert its standard output
			below the cursor or the specified line.  A temporary
			file is used to store the output of the command which
			is then	read into the buffer.  'shellredir' is used to
			save the output of the command, which can be set to
			include stderr or not.  {cmd} is executed like with
			":!{cmd}", any '!' is replaced with the previous

> 			command |:!|. See |++opt| for the possible values of
> 			[++opt].

Reviewer 2: ```suggestion
			command |:!|.
			See |++opt| for the possible values of [++opt].
```

							*:r:* *:read:*
:[range]r[ead] :{cmd}
			Execute Ex command {cmd} and insert its output below

> 			the cursor or the specified line

Reviewer 2: ```suggestion
			the cursor or the specified line.
```
